coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/core/ProposalDevelopmentViewHelperServiceImpl.java

```diff
@@ -351,6 +351,10 @@ protected boolean performDeleteLineValidation(ViewModel model, String collection
                 getProposalDevelopmentPermissionsService().processDeletePermission(document, ((ProposalUserRoles) deleteLine));
             }
         }
+        
+        if(deleteLine instanceof ProposalDevelopmentAttachment) {
+        	((ProposalDevelopmentAttachment)deleteLine).setUpdated(true);
+        }
```

Comment:
If we are deleting the attachment do we need to worry about this? Isn't it just removed?